packages/attest/package.json

```diff
@@ -46,9 +46,15 @@
     "@actions/core": "^1.10.1",
     "@actions/github": "^6.0.0",
     "@actions/http-client": "^2.2.1",
+    "@octokit/plugin-retry": "^6.0.1",
     "@sigstore/bundle": "^2.3.0",
     "@sigstore/sign": "^2.3.0",
     "jsonwebtoken": "^9.0.2",
     "jwks-rsa": "^3.1.0"
+  },
+  "overrides": {
+    "@octokit/plugin-retry": {
+      "@octokit/core": "^5.2.0"
+    }
```

Comment:
The `@octokit/plugin-retry` package has a very loose peer dependency on `@octokit/core`, however we need to pin it to the same version which is used by `@actions/github` to avoid issues with type inconsistencies.